ui/component/header/index.js

```diff
@@ -1,40 +1,31 @@
-import * as SETTINGS from 'constants/settings';
 import { connect } from 'react-redux';
```

Comment:
It is clipping on my phone.

This screenshot is from emulator, but looks similar on my phone:
![image](https://user-images.githubusercontent.com/64950861/146149242-b776e894-44e2-402c-85ce-d5e7477ca6d8.png)

Reply:
It's closer now, but not totally clipping as it did in my screenshot. I thought it was a feature! Lol

Reply:
<img src="https://user-images.githubusercontent.com/64950861/146505156-4512dab4-5a04-4c37-bf63-3fb015e3d141.jpg" width="300">

Reply:
It it still clipping for me on Firefox Android, but also just noticed that Chrome Android does not clip.  Chrome Android also handles the "PRs in this dev instance" bubble better, where the text doesn't overflow the button/

Reply:
oh thank you that's probably why. been testing on chrome all the time